src/client/src/error.rs

```diff
@@ -7,12 +7,14 @@ use crate::config::ConfigError;
 use semver::Version;
 
 /// Possible errors that may arise when using Fluvio
+#[allow(clippy::upper_case_acronyms)]
```

Comment:
We added `-A clippy::upper_case_acronyms` to the `make check-clippy` task, so I think you shouldn't need to have these to pass CI now

Reply:
I agree.

Reply:
Removed